src/octoprint/static/js/app/viewmodels/terminal.js

```diff
@@ -338,6 +338,14 @@ $(function() {
             copyToClipboard(lines.join("\n"));
         };
 
+        self.clearLogWidow = function() {
```

Comment:
Wouldn't it make more sense to always clear both the plain and fancy version of the log? Otherwise things could get weird if you swap between the two...

Reply:
Indeed it would - Fixed!